tests/tests_pytorch/callbacks/test_stochastic_weight_avg.py

```diff
@@ -255,3 +281,103 @@ def on_train_epoch_start(self):
     )
     trainer.fit(model)
     assert model.on_train_epoch_start_called
+
+
+def _swa_resume_training_from_checkpoint(tmpdir, model, resume_model, ddp=False):
+    swa_start = 3
+    trainer_kwargs = {
+        "default_root_dir": tmpdir,
+        "max_epochs": 5,
+        "accelerator": "cpu",
+        "strategy": "ddp_spawn_find_unused_parameters_false" if ddp else None,
+        "devices": 2 if ddp else 1,
+        "limit_train_batches": 5,
+        "limit_val_batches": 0,
+        "accumulate_grad_batches": 2,
+        "enable_progress_bar": False,
+    }
+    trainer = Trainer(callbacks=SwaTestCallback(swa_epoch_start=swa_start, swa_lrs=0.1), **trainer_kwargs)
+
+    with _backward_patch(trainer), pytest.raises(Exception if ddp else DummyError):
+        trainer.fit(model)
+
+    checkpoint_dir = Path(tmpdir) / "lightning_logs" / "version_0" / "checkpoints"
```

Comment:
is this correct?
I remember it should be just `Path(tmpdir) / "checkpoints"`

Reply:
Yes this does seem to be correct. This matches `trainer.checkpoint_callback.dirpath`, except that is None when using DDP so I can't use that property and have to specify the path here.

Reply:
Yes that's correct, dirpath wouldn't be available, so this here looks fine to me.

FYI, `checkpoint_callback.best_model_path` would however be available, as we specifically restore it when returning to the main process. You can try it to save a few lines but I think the current explicit way is good for the test!